api-client/src/main/java/com/xing/api/data/SafeCalendar.java

```diff
@@ -126,4 +126,17 @@ protected void complete() {
             super.complete();
         }
     }
+
+    @Override
+    public boolean equals(Object obj) {
+        return super.equals(obj)
+              && ((SafeCalendar) obj).isSet(YEAR) == isSet(YEAR)
+              && ((SafeCalendar) obj).isSet(MONTH) == isSet(MONTH)
+              && ((SafeCalendar) obj).isSet(DAY_OF_MONTH) == isSet(DAY_OF_MONTH);
+    }
+
+    @Override
+    public int hashCode() {
+        return super.hashCode();
```

Comment:
I'd better implement a proper hashcode like you did with equals or just remove this method as it does nothing.